lib/common/Memory/ArenaAllocator.cpp

```diff
@@ -447,8 +447,8 @@ ReleaseHeapMemory()
     }
 }
 
-template __forceinline char *ArenaAllocatorBase<InPlaceFreeListPolicy, 0, 0, 0>::AllocInternal(size_t requestedBytes);
-template __forceinline char *ArenaAllocatorBase<InPlaceFreeListPolicy, 3, 0, 0>::AllocInternal(size_t requestedBytes);
+template CH_FORCE_INLINE char *ArenaAllocatorBase<InPlaceFreeListPolicy, 0, 0, 0>::AllocInternal(size_t requestedBytes);
+template CH_FORCE_INLINE char *ArenaAllocatorBase<InPlaceFreeListPolicy, 3, 0, 0>::AllocInternal(size_t requestedBytes);
```

Comment:
We can revert these CH_FORCE_INLINE changes with the defining __forceinline proposal.